README.md

```diff
@@ -1,14 +1,16 @@
-# Protobuf support for Dart
-
 [![CI status](https://github.com/google/protobuf.dart/workflows/Dart%20CI/badge.svg)](https://github.com/google/protobuf.dart/actions?query=workflow%3A%22Dart%22+branch%3Amaster)
-[![pub package](https://img.shields.io/pub/v/protobuf.svg)](https://pub.dev/packages/protobuf)
+
+## Protobuf support for Dart
 
 [Protocol Buffers](https://developers.google.com/protocol-buffers) (protobuf)
 are Google's language-neutral, platform-neutral, extensible mechanism for
 serializing structured data.
 
-This repository is home to packages related to [protobuf for Dart](https://pub.dev/documentation/protobuf/latest/).
-It contains:
+This repository is home to packages related to
+[protobuf for Dart](https://pub.dev/documentation/protobuf/latest/).
+
+## protobuf
+[protobuf](protobuf/) - a support library for the generated code
 
- - [protoc_plugin](protoc_plugin/) A Dart back-end for the protoc compiler.
- - [protobuf](protobuf/) Support library for the generated code.
+## protoc_plugin
+[protoc_plugin](protoc_plugin/) - a Dart back-end for the protoc compiler
```

Comment:
"protobuf" and "protoc_plugin" should be smaller than the repo description I think. Maybe make "Protobuf support for Dart" h1, or "protobuf" and "protoc_plugin" h3?

I'm actually not sure if we need headers given that they don't have much text under, I like the current version. Is this a template we use for Google repos?

Reply:
There isn't really a standard we use for monorepos. I'm going to re-work this however to use a markdown table and see how that looks.